test_search.py

```diff
@@ -1,11 +1,9 @@
 import time
 import pytest
-
 from selenium.common.exceptions import NoSuchElementException
 from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support.select import Select
-
```

Comment:
These spaces should still be here. See, https://www.python.org/dev/peps/pep-0008/#imports